portal-impl/src/com/liferay/mail/messaging/MailMessageListener.java

```diff
@@ -47,6 +49,13 @@
 	protected void doMailMessage(MailMessage mailMessage) throws Exception {
 		InternetAddress from = filterInternetAddress(mailMessage.getFrom());
 
+		String smtpUserName =
```

Comment:
Why not ask customers to add the from address (which is configurable) to the O365 list of allowed users? Overriding the from address configuration with the smtpUserName is probably the wrong choice for all configurations except O365.

I'd prefer to document this as an extra configuration step if people want to use O365.

Reply:
Hi @adolfopa this is a good point.

Reply:
I haven't found a way yet to add another sender with a different domain (liferay.com) to an outlook personal account.
With a commercial (trial) Office365 account, I could not connect to the SMPT server.